core/templates/pages/contributor-dashboard-page/login-required-message/login-required-message.component.html

```diff
@@ -1,22 +1,22 @@
 <div class="oppia-login-required-message-container">
   <div class="oppia-mascot-avatar-container">
-    <a ng-class="{'oppia-disabled-image-link': ($ctrl.OPPIA_AVATAR_LINK_URL === null)}"
-       ng-href="<[$ctrl.OPPIA_AVATAR_LINK_URL]>"
+    <a ngClass="{'oppia-disabled-image-link': (OPPIA_AVATAR_LINK_URL === null)}"
```

Comment:
Why is it `'oppia-disabled-image-link': (OPPIA_AVATAR_LINK_URL === null)` and not `'oppia-disabled-image-link': OPPIA_AVATAR_LINK_URL === null`. Any reason for the extra set of parentheses.

Reply:
The parentheses were unnecessary, I removed them.
I was also missing the square brackets around `ngClass`, I added them and re-tested everything.